flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/spargel/MessagingFunction.java

```diff
@@ -314,7 +314,7 @@ public void remove() {
 
 	/**
 	 * Retrieves the vertex in-degree (number of in-coming edges).
-	 * @return The in-degree of this vertex if the {@link IterationConfiguration#setOptDegrees(boolean)}
+	 * @return The in-degree of this vertex
```

Comment:
There is no IterationConfiguration#setOptDegrees(boolean) any more and if you look at the source code which this javadoc relates to it. It looked like copy-paste comment from somewhere else.